lib/vanagon/platform/windows.rb

```diff
@@ -85,7 +85,27 @@ def generate_nuget_package(project)
       # @param project [Vanagon::Project] project to name
       # @return [String] name of the windows package for this project
       def nuget_package_name(project)
-        "#{project.name}-#{@architecture}-#{project.version}.#{project.release}.nupkg"
+        "#{project.name}-#{@architecture}-#{nuget_package_version(project)}.nupkg"
+      end
+
+      # Nuget versioning is awesome!
```

Comment:
It's actually not the worst. It's probably a good thing that we're being confined to semver